sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -293,6 +293,7 @@ object RapidsReaderType extends Enumeration {
 }
 
 object RapidsConf {
+  val MULTITHREAD_READ_NUM_THREADS_DEFAULT = 20
```

Comment:
nit, Can we move this to be right above the `MULTITHREAD_READ_NUM_THREADS` definition below?